scripts/saveCanvas.js

```diff
@@ -0,0 +1,24 @@
+/**
+  Fetch the elements within the canvas then, using _serializeCanvas,
+  create a string represneting that data. Store the resulting string
+  at localStorage.set(storageName).
+
+   @param {string} storageName - a key for a value in localStorage
+    used to store the data.
+*/
+window.saveCanvas = function saveCurrentCanvas(storageName) {
```

Comment:
I'm a little confused about using this function, do you just call it by typing
window.saveCanvas('file-name-here')    ?